### PR TITLE
feat: added databox docpage

### PR DESCRIPTION
**Pull request by calebjclark**
A static docpage is automatically created and packaged into Dbx files. 

Databox Core serves the documentation at /databox/versionHash

The docpages are currently worthless until we publish @ulixee/stream, @ulixee/sql, or release the postgres layer.

### CODE REVIEW

File: databox/docpage/src/App.vue [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+            <option value="sql">Ulixee SQL</option>
+            <option value="stream">Ulixee Stream</option>
+            <option value="postgres">Postgres</option>
+          </select>
+        </h2>
+        <Prism language="javascript" v-if="exampleType === 'sql'">
+          import Sql from '@ulixee/sql';
+
+          (async function () {
+            const stream = new Sql('ulx://');
```
[the remark this answers is not in the document]
Comment (blakebyrnes) [reply databox/docpage/src/App.vue:39]: I'm not sure I see how examples are injected. Is that working?

File: package.json [hunk 2]
```diff
@@ -6,11 +6,13 @@
...
     "build:hero": "cd hero && yarn build",
-    "build:all": "yarn build:hero && yarn build",
+    "build:chromealive-extension": "cd apps/chromealive-extension && yarn build",
+    "build:chromealive": "cd apps/chromealive && yarn build",
+    "build:all": "yarn build:hero && yarn build:chromealive && yarn build",
     "build:dist": "shx rm -rf build-dist && tsc -b tsconfig.dist.json && yarn copy:dist && yarn workspace @ulixee/apps-chromealive-extension build:dist",
     "copy:build": "ulx-repo-after-build && ulx-repo-adjust-paths",
     "copy:dist": "ulx-repo-after-build --dist",
-    "tsc": "tsc -b -i tsconfig.json && yarn copy:build",
+    "tsc": "tsc -b -i tsconfig.json && yarn copy:build && yarn workspace @ulixee/databox-docpage build",
```
[the remark this answers is not in the document]
Comment (calebjclark) [reply package.json]: Agreed. Fixed.

File: databox/docpage/src/App.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (calebjclark) [reply databox/docpage/src/App.vue:39]: Not yet.

[the remark this answers is not in the document]
Comment (calebjclark) [reply databox/docpage/src/App.vue:39]: We should fix those in a later PR once Stream/SQL is released.

**Pull request merged**